src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    void process(FruitTransaction transaction, FruitStorage storage);
```

Comment:
Pass fruitStorage to impl constructor
```suggestion
    void process(FruitTransaction transaction);
```